tests/api_app/analyzers_manager/file_analyzers/test_lnk_info.py

```diff
@@ -0,0 +1,58 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+
+from django.core.files import File
+
+from api_app.analyzers_manager.file_analyzers.lnk_info import LnkInfo
+from api_app.analyzers_manager.models import AnalyzerConfig
+from api_app.models import Job
+from tests import CustomTestCase
+
+
+class LnkInfoTestCase(CustomTestCase):
+    fixtures = [
+        "api_app/fixtures/0001_user.json",
+    ]
+
+    def _create_job(self, sample, mimetype, analyzer_config) -> Job:
+        try:
+            with open(sample, "rb") as f:
+                _job = Job.objects.create(
+                    is_sample=True,
+                    file_name=sample,
+                    file_mimetype=mimetype,
+                    file=File(f),
+                    user=self.superuser,
+                )
+                _job.analyzers_to_execute.set([analyzer_config])
+                return _job
+        except Exception as e:
+            print(f"Error: {e}")
+
+    def _analyze(self, sample_name, sample_md5, sample_mimetype):
+        analyzer_config = AnalyzerConfig.objects.get(name="Lnk_Info")
+        lnk_info_analyzer = LnkInfo(analyzer_config)
+        lnk_info_analyzer.md5 = sample_md5
+        lnk_info_analyzer.filename = f"test_files/{sample_name}"
+        lnk_info_analyzer.file_mimetype = sample_mimetype
+        job = self._create_job(
+            f"test_files/{sample_name}", sample_mimetype, analyzer_config
+        )
+        lnk_info_analyzer.start(job.id, {}, 1)
+        return lnk_info_analyzer.report.report
+
```

Comment:
to reduce the code a little, please use a class in  the middle that has _create_job and _analyze implemented so we don't have the same code in all the tests